src/environment/mod.rs

```diff
@@ -203,6 +204,21 @@ fn setup_env_for_git_repo() -> GitInfo {
     git_info_clone
 }
 
+fn setup_env_for_execution() -> ExecutionInfo {
+    // Set if we are in CI or not.
+    let is_ci_var_value = if executioninfo::is_ci() {
+        "TRUE"
+    } else {
+        "FALSE"
+    };
+    env::set_var("CARGO_MAKE_IS_CI", is_ci_var_value);
```

Comment:
I really like this one. very helpful for people writing scripts and adding conditions. cool!!!
can we rename it to CARGO_MAKE_CI?
basically i renamed everything with prefixes like
CARGO_MAKE_CRATE_xxx
CARGO_MAKE_RUST_xxxxx
and so on...
so CARGO_MAKE_CI and CARGO_MAKE_CI_xxx would be more consistent.